aggregations/db_tables/near_ecosystem_entities.py

```diff
@@ -14,7 +14,8 @@ def sql_create_table(self):
                 website     TEXT, 
                 category    TEXT,    
                 status      TEXT, 
-                contract    TEXT
+                contract    TEXT,
+                logo        TEXT
```

Comment:
We don't recreate all the tables every day. Instead of just altering the table definition, you should use ALTER TABLE syntax and add the column. Or write the code that will handle the migration in some other way (e.g. you may drop the table and recreate it, which is going to happen every hour, so that is not really a great path forward, just an example).